src/main/java/com/teamrocket/tms/services/task/TaskServiceValidationImpl.java

```diff
@@ -28,7 +28,7 @@ public void validateTaskAlreadyExists(TaskDTO taskDTO) {
 
     @Override
     public void validateTaskCanBeAssigned(Task task) {
-        if(task.isComplete() == true || task.getUser() != null){
+        if(task.isComplete() || task.getUser() != null){
```

Comment:
Please don't forget to format the code. Between `){` should be a space, that's a sign that the class wasn't formatted. 
Please consider formatting each class updated.

Reply:
Thanks for the review! Just formatted all classes and went through the changes.